packages/inertia-react/index.d.ts

```diff
@@ -86,3 +86,29 @@ export interface InertiaFormProps<TForm = Record<string, any>> {
 }
 
 export function useForm<TForm = Record<string, any>>(initialValues: TForm): InertiaFormProps<TForm>;
+
+export interface AppSetupPageProps {
+    initialPage: Inertia.PageProps,
+    initialComponent: any,
+    resolveComponent: (name: string) => Promise<any>,
+    titleCallback: (title: string) => string,
+    onHeadUpdate: ((elements: any[]) => boolean) | null,
+}
+
+export interface AppSetupProps {
+    el: HTMLElement | null
```

Comment:
```suggestion
    el: HTMLElement
```
Think this doesn't have to be null